app/gui2/src/appRunner.ts

```diff
@@ -11,6 +11,14 @@ async function runApp(
   _metadata?: object | undefined,
   pinia?: Pinia | undefined,
 ) {
+  const ignoreParamsRegex = (() => {
```

Comment:
We use the `_metadata` object here to pass an option that specifies an ignore mask